packages/ia-topnav/src/data/menus.js

```diff
@@ -210,7 +210,7 @@ const software = (baseHost = prodHost) => ({
     title: 'All software',
```

Comment:
Q: we had a plan to move this output and feed the data to the web-component. is that in the pipeline?

Reply:
The plan was to have ia-topnav used as nav for all users, allowing me to remove mobile-top-nav and the nav building in Nav.inc. At that point, Nav.inc would be updated to simply send its subnav arrays to ia-topnav and it would blindly render whatever nav items you fed it.

Reply:
ah, cool. so that will happen once mobile nav is gone. dope!